src/core/repoIdRegistry.js

```diff
@@ -14,6 +14,7 @@ const REPO_ID_REGISTRY_COMPAT = {type: "REPO_ID_REGISTRY", version: "0.2.0"};
 
 export type RegistryEntry = {|
   +repoId: RepoId,
+  +timestamp: Date,
```

Comment:
JS date objects scare me, so I'd likely prefer timestamp in ms since epoch, as retrievable from [Date.now](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/now). So as to be unambiguous, the field can be called `timestampMsSinceEpoch`. 
Pinging @wchargin  in case he has another opinion

Reply:
Okay, sounds good, I'll change it in a bit.

Reply:
Eh… JS `Date` objects certainly aren’t great\*, but I don’t see any
reason to be scared of them. Just treat them as if they were immutable
and you should be okay. I have a moderate preference for `Date` over
`timestampMsSinceEpoch`: `Date` encapsulates the units perfectly well.

\* Primarily: they’re mutable, and `getDay()` does not get the day.